domain/induction/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/induction/Induction.kt

```diff
@@ -28,10 +28,10 @@ data class Induction(
    */
   val previousQualifications: PreviousQualifications?,
   /**
-   * Any additional training that the Prisoner may have done previously. Null if the Prisoner has not been asked about
-   * their additional training.
+   * Any additional training that the Prisoner may have done previously. Mandatory as the Prisoner is always asked about
+   * it.
    */
-  val previousTraining: PreviousTraining?,
+  val previousTraining: PreviousTraining,
```

Comment:
On reflection, I originally made the domain too flexible. For the purposes of mapping, particularly when mapping two separate entities (e.g. eduction and training) into one response object, it's considerably easier if one of them is guaranteed to be populated!